src/MessageStrip/MessageStrip.js

```diff
@@ -48,10 +48,10 @@ const MessageStrip = (props) => {
         className
     );
 
-    const alertId = otherProps?.id || shortId.generate();
+    const alertId = otherProps?.id || useUniqueId();
```

Comment:
We can't [conditionally call hooks](https://reactjs.org/docs/hooks-rules.html) like this, or with ternary expressions like others.

We can split it onto separate lines though,
```js
    const generatedId = useUniqueId();
    const alertId = otherProps?.id || generatedId;
```